### PR TITLE
[ignore this - everything on fe branch, dunno how to delete] hashing

### PR DESCRIPTION
**Pull request by zhangrmatthew**

### CODE REVIEW

**Review by kdduan — DISMISSED** [review]
LGTM! 🚀

**Comment by cindykimxp** [comment]
make sure to resolve conflicts and then merge! looks good :)

**Review by janethuangg — CHANGES_REQUESTED** [review]
as mentioned over slack, check out the GET endpoint to figure out how to incorporate passport auth into this – key changes include 
using `const userId = req.user._id` in the controller file 
and 
adding `passport.authenticate("jwt", { session: false }),` into the route

File: back-end/routes/userRoutes.js [hunk 1]
```diff
@@ -5,16 +5,23 @@ const userController = require("../controllers/userController");
 
 router.post("/", userController.createUser);
 
-router.delete("/:userId", userController.deleteUser);
+router.delete(
+  "/:userId",
```
Comment (janethuangg) [inline back-end/routes/userRoutes.js:9]: route should just be `"/"`

File: back-end/routes/userRoutes.js [hunk 2]
```diff
@@ -5,16 +5,23 @@ const userController = require("../controllers/userController");
...
 router.get(
   "/",
   passport.authenticate("jwt", { session: false }),
   userController.getUser
 );
 
-// TODO: update this to align with get request (get id from req.user not req.params)
-router.patch("/:userId", userController.updateUser);
+router.patch(
+  "/:userId",
```
Comment (janethuangg) [inline back-end/routes/userRoutes.js:21]: route should just be `"/"`

**Review by kdduan — CHANGES_REQUESTED** [review]
the BE logic looks good - main change is to remove the userId url param from the endpoint since the FE won't have access to the raw userId

**Pull request closed without merging**